### PR TITLE
Collected small changes and fixes

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

The pull request contains multiple small changes and fixes.

**Related Issue(s)**

This addresses issues about too opaque errors discussed on the LAMMPS slack.

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

The following individual changes are included:
- flag develop branch as development version
- add more information about the release procedure
- add new utils::errorurl() calls to the variable command explaining that extra arguments can come from missing quotes or excess whitespace
- provide access to the external force from fix external to the LAMMPS script interface and dumps.

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system
- [x] The feature has been verified to work with the conventional build system
- [x] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

**Further Information, Files, and Links**

### CODE REVIEW

File: doc/src/fix_external.rst [hunk 1]
```diff
@@ -173,9 +173,17 @@ stress/atom <compute_stress_atom>` commands.  The former can be
...
+This fix computes a global scalar, a global vector, and a per-atom array
+which can be accessed by various :doc:`output commands <Howto_output>`.
+The scalar is the potential energy discussed above.  The scalar stored
+by this fix is "extensive".  The vector is a custom vector set by the
+external program using the :cpp:func:`lammps_fix_external_set_vector()
+<lammps_fix_external_set_vector>` and
+:cpp:func:`lammps_fix_external_set_vector_length()
+<lammps_fix_external_set_vector_length>` calls of the LAMMPS library
+interface or the equivalent call of the Python or Fortran modules. The
+per-atom array has 3 column for each atom and is the applied external
```
Comment (sjplimp) [inline doc/src/fix_external.rst]: 3 values for each atom

File: doc/src/fix_external.rst [hunk 2]
```diff
@@ -173,9 +173,17 @@ stress/atom <compute_stress_atom>` commands.  The former can be
 accessed by :doc:`thermodynamic output <thermo_style>`.  The default
 setting for this fix is :doc:`fix_modify virial yes <fix_modify>`.
 
-This fix computes a global scalar which can be accessed by various
-:doc:`output commands <Howto_output>`.  The scalar is the potential
-energy discussed above.  The scalar stored by this fix is "extensive".
+This fix computes a global scalar, a global vector, and a per-atom array
+which can be accessed by various :doc:`output commands <Howto_output>`.
+The scalar is the potential energy discussed above.  The scalar stored
+by this fix is "extensive".  The vector is a custom vector set by the
```
Comment (sjplimp) [inline doc/src/fix_external.rst]: I think you mean a global vector, not a custom vector?  The latter is a per-atom property.

**Comment by sjplimp** [comment]
Also, in the fix external doc page there is a pgraph which starts:
These allow to set per-atom energy ...
It would read better as
These enable setting per-atom energy ... and the length and individual values ...

File: .github/release_steps.md [hunk 3]
```diff
@@ -1,28 +1,78 @@
 # LAMMPS Release Steps
 
-The following notes chronicle the current steps for preparing and publishing LAMMPS releases.
-For definition of what LAMMPS versions and the different kinds of releases mean, please
-refer to [the corresponding section in the LAMMPS manual](https://docs.lammps.org/Manual_version.html).
+The following notes chronicle the current steps for preparing and publishing LAMMPS releases.  For
+definition of what LAMMPS versions and the different kinds of releases mean, please refer to [the
```
Comment (jtclemm) [inline .github/release_steps.md]: Could shorten "for definitions of different LAMMPS versions and releases"

**Review by jtclemm — APPROVED** [review]

**Review by sjplimp — APPROVED** [review]

**Pull request merged**